doc/Yang Model Guidelines.md

```diff
@@ -1,20 +1,23 @@
-#   YANG MODEL GUIDELINES
+#   SONiC YANG MODEL GUIDELINES
 
 ### Rev 1.0 ###
 
 ### Revision ###
 
  | Rev |     Date    |       Author       | Change Description                |
  |:---:|:-----------:|:------------------:|-----------------------------------|
- | 1.0 | 22 Aug 2019             |      Praveen Chaudhary      | Initial version                   |
+ | 1.0 | 22 Aug 2019             |      Praveen Chaudhary      | Initial version                    |
```

Comment:
We have not released "1.0" major version yet. Should we change to 0.1, 0.2 etc.

Reply:
Agree. 
As this is not committed yet, you may not increment for adding comments. 
It can be 0.1
Subsequent PRs can increment.